examples/apps/cra-demo/README.md

```diff
@@ -104,40 +106,45 @@ const getContainerId = () => {
 };
 ```
 
-### 3.b Instantiate a service client instance
+### 3.b Configure the service client
 
-`TinyliciousClient` instance needs to be instantiated before use and can take an optional configuration object to changes settings such as default port. Depending on your deploy target, you might create and use different client packages.
+In this demo we will only be dealing with a local configuration, but in a production environment you could swap this out with a production configuration without changing your application.
 
 ```js
 // add below getContainerId
-const tinyliciousClient = new TinyliciousClient();
+const localConfig = {
+    tenantId: "local",
+    tokenProvider: new InsecureTokenProvider("tenantId", { id: "userId" }),
+    orderer: "http://localhost:7070",
+    storage: "http://localhost:7070",
+};
 ```
 
 ## 4. <a style="position: relative; top: 20px" name="init"></a> Get Fluid Data
 
-Before we can access any Fluid data, we need to make a call to the `TinyliciousClient` with necessary service configuration and container schema.
+Before we can access any Fluid data, we need to create our configured FRS `client` and define our container schema.
 
-- `serviceConfig` is going to vary per service. With Tinylicious it will only requires an `id`.
 - `containerSchema` is going to include a string `name` and a collection of the data types our application will use.
 
 The following `getFluidData` function utilizes the `getContainerId` to return a unique ID and determine if this is an existing document (`getContainer`) or if we need to create a new one (`createContainer`).
 
-Since this function is an async, we'll need to wait for the `initialObjects` to be returned. Once returned, each `initialObjects` key will point to a connected data structure as per defined in the schema.
+Since this `getFluidData` function is an async, we'll need to wait for the `initialObjects` to be returned. Once returned, each `initialObjects` key will point to a connected data structure as defined in the schema.
```

Comment:
```suggestion
Since `getFluidData` is an async function, we'll need to `await` for the `initialObjects` to be returned. Once returned, each `initialObjects` key will point to a connected data structure as defined in the schema.
```